### PR TITLE
docs: Add our custom `x/*` modules to Swagger UI

### PR DESCRIPTION
**Pull request by youngjoon-lee**
As described in the [protoc-swagger-gen.sh](https://github.com/medibloc/panacea-core/blob/266cf776e1aea78fc12ba8d1b2192abba7c6b6cd/scripts/protoc-swagger-gen.sh#L27), our custom `x/*` modules should be added to the `client/docs/config.json`.

image

### CODE REVIEW

File: client/docs/swagger-ui/swagger.yaml [hunk 1]
```diff
@@ -29515,6 +29515,881 @@ paths:
                       format: byte
       tags:
         - Query
+  '/panacea/aol/v2/owners/{owner_address}/topics':
```
Comment (gyuguen) [inline client/docs/swagger-ui/swagger.yaml:29518]: Is auto generation?

Comment (youngjoon-lee) [reply client/docs/swagger-ui/swagger.yaml:29518]: Yes. Everything in the `client/docs/` are auto-generated by [make proto-swagger-gen](https://github.com/medibloc/panacea-core/blob/83b3983ba35b7be47d296402ed29fdd3fb0a2269/Makefile#L51). That command invokes the [protoc-swagger-gen.sh](https://github.com/medibloc/panacea-core/blob/83b3983ba35b7be47d296402ed29fdd3fb0a2269/scripts/protoc-swagger-gen.sh#L28) script.

That script has two big steps:
1. Generating Swagger docs from all proto files
2. Combining them into one Swagger doc (using `swagger-combine`)

Comment (gyuguen) [reply client/docs/swagger-ui/swagger.yaml:29518]: OK. Thank you!

**Review by gyuguen — APPROVED** [review]
👍

**Pull request merged**